src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -213,94 +220,74 @@ def resolve_dependencies(self, main_ctx: AnalyzeContext) -> dict[str, Dependency
-                    if isinstance(main_ctx.dynamic_data["build_spec"]["tool"], Maven):
-                        logger.info(
-                            "Running %s version %s dependency analyzer on %s",
-                            tool_name,
-                            tool_version,
-                            main_ctx.repo_path,
-                        )
-
-                        log_path = os.path.join(
-                            global_config.build_log_path,
-                            f"{main_ctx.repo_name}.{tool_name}.log",
-                        )
-
-                        commands = dep_analyzer.get_cmd()
-                        # We are using this flag because we are running
-                        # mvnw in the macaron root path, not in the target repo directory.
-                        commands.extend(["-f", main_ctx.repo_path])
-                        # Suppressing Bandit's B603 report because the repo paths are validated.
-                        analyzer_output = subprocess.run(  # nosec B603
-                            commands,
-                            stdout=subprocess.PIPE,
-                            stderr=subprocess.STDOUT,
-                            check=True,
-                            cwd=global_config.resources_path,
-                            timeout=defaults.getint("dependency.resolver", "timeout", fallback=1200),
-                        )
-                        with open(log_path, mode="w", encoding="utf-8") as log_file:
-                            logger.info("Storing dependency resolver log to %s", log_path)
-                            log_file.write(analyzer_output.stdout.decode("utf-8"))
-
-                        deps_resolved = dep_analyzer.collect_dependencies(main_ctx.repo_path)
-                    else:
-                        logger.info(
-                            "Dependency analyzer is not available for %s",
-                            main_ctx.dynamic_data["build_spec"]["tool"].name,
-                        )
-                else:
-                    logger.error(
-                        "Please remove pom.xml file in %s.",
-                        global_config.macaron_path,
-                    )
+                with open(log_path, mode="a", encoding="utf-8") as log_file:
+                    log_file.write(analyzer_output.stdout.decode("utf-8"))
 
             except (subprocess.CalledProcessError, subprocess.TimeoutExpired) as error:
                 logger.error(error)
-                with open(log_path, mode="w", encoding="utf-8") as log_file:
-                    logger.info("Storing dependency resolver log to %s", log_path)
+                with open(log_path, mode="a", encoding="utf-8") as log_file:
                     log_file.write(error.output.decode("utf-8"))
             except FileNotFoundError as error:
                 # Only happen if the gradlew at the repo dir has an invalid format
```

Comment:
We are now using the gradlew and mvnw directly from our `resources` directory (previously, we copied the wrapper files to the repository directory to run it) so I think this comment could be removed.